### PR TITLE
Bugfix/platfowner/bugfix

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Upgrade nodejs version to v12 from v10 to use --async-stack-traces option 
- Allow chain segment merging while in SERVING (returning to the original logic) 
- Add enableVersionRenaming feature flag (for further debugging) 
- Log stderr logs to error_logs.txt 
- Tweak logging messages

Background:
- To inspect #171 , tried the followings:
  1) Revert effective logical changes since release/R27
  2) Improve logging including nodejs version upgrade to use --async-stack-traces option (see https://stackoverflow.com/questions/55162619/why-do-i-lose-stack-trace-when-using-async-await-in-node-js) 
- As a result, 1) made no changes but it seems that the issue is gone by 2) (not sure, let take a longer look) 

Related memo: https://www.notion.so/comcom/Stability-debugging-38a0c95751a340038dbb5efb013dd867

#171

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -137,9 +141,11 @@ class BlockchainNode {
     if (!this.stateManager.finalizeVersion(newFinalVersion)) {
       logger.error(`[${LOG_HEADER}] Failed to finalize version: ${newFinalVersion}`);
     }
-    logger.info(`[${LOG_HEADER}] Renaming version: ${version} -> ${newFinalVersion}`);
-    if (!this.stateManager.renameVersion(version, newFinalVersion)) {
-      logger.error(`[${LOG_HEADER}] Failed to replace version: ${version} -> ${newFinalVersion}`);
+    if (FeatureFlags.enableVersionRenaming) {
+      logger.info(`[${LOG_HEADER}] Renaming version: ${version} -> ${newFinalVersion}`);
+      if (!this.stateManager.renameVersion(version, newFinalVersion)) {
+        logger.error(`[${LOG_HEADER}] Failed to replace version: ${version} -> ${newFinalVersion}`);
```
Comment (liayoo) [inline node/index.js:147]: rename?

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks! I'll take a look at the issue as well.

File: node/index.js (hunk 1 above)
Comment (platfowner) [reply node/index.js]: fixed. thanks.

**Comment by platfowner** [comment]
Thanks for the fast review. Now merging..

**Pull request merged**